modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -138,10 +136,6 @@ def get_pdf_data
             data: {}
           }
         end
-
-        def benefits_doc_api
-          ClaimsApi::BD.new
-        end
       end
```

Comment:
not being used

Reply:
we have access to this in the application controller in v2 anywho. same method name and all, so even if we were to use it, this is just duplication.